g1ant-language/language-basics.md

```diff
@@ -0,0 +1,321 @@
+# Basics
+
+When you learn new language, you always start with basics: pronunciation, spelling, fundamental vocabulary, general rules of constructing sentences and expressing your thoughts. The same applies to any programming language, where you learn its commands, arguments, syntax and all the rules that make it logical.
+
+G1ANT.Language is no different, so before you start creating robots, you must know the foundations, on which you can build your scripts. We will guide you through language basics and present you with the elementary tools, which help you in writing your code.
+
+## Workspace
+
+G1ANT.Robot program offers you powerful and convenient tools to help you master all aspects of programming in G1ANT.Language.
```

Comment:
SUGGESTION: ..all aspects of programming in G1ANT.Language **to robotise and automate common and repetitive tasks**.

Reply:
```suggestion
G1ANT.Robot program offers you powerful and convenient tools to help you master all aspects of programming in G1ANT.Language to robotize and automate common and repetitive tasks.
```